runtime/src/bank.rs

```diff
@@ -5158,10 +5174,50 @@ impl Bank {
         );
         check_time.stop();
 
+        let sanitized_output = self.load_and_execute_sanitized_transactions(
+            sanitized_txs,
+            &mut check_results,
+            &mut error_counters,
+            enable_cpi_recording,
+            enable_log_recording,
+            enable_return_data_recording,
+            timings,
+            account_overrides,
+            log_messages_bytes_limit,
+            &mut check_time,
+        );
+        LoadAndExecuteTransactionsOutput {
+            loaded_transactions: sanitized_output.loaded_transactions,
+            execution_results: sanitized_output.execution_results,
+            retryable_transaction_indexes,
+            executed_transactions_count: sanitized_output.executed_transactions_count,
+            executed_non_vote_transactions_count: sanitized_output
+                .executed_non_vote_transactions_count,
+            executed_with_successful_result_count: sanitized_output
+                .executed_with_successful_result_count,
+            signature_count: sanitized_output.signature_count,
+            error_counters,
+        }
+    }
+
+    #[allow(clippy::too_many_arguments)]
+    fn load_and_execute_sanitized_transactions(
+        &self,
+        sanitized_txs: &[SanitizedTransaction],
+        check_results: &mut [TransactionCheckResult],
+        error_counters: &mut TransactionErrorMetrics,
+        enable_cpi_recording: bool,
+        enable_log_recording: bool,
+        enable_return_data_recording: bool,
+        timings: &mut ExecuteTimings,
+        account_overrides: Option<&AccountOverrides>,
+        log_messages_bytes_limit: Option<usize>,
+        check_time: &mut Measure,
```

Comment:
we can probably not pass it in as parameter. It's only being logged, and accumulated in `timings`. That can be done in the caller.